.github/workflows/engine-benchmark.yml

```diff
@@ -13,8 +13,8 @@ on:
         type: boolean
         default: false
 jobs:
-  benchmark-engine:
-    name: Benchmark Engine
+  benchmark-engine-graal-vm-ce:
```

Comment:
Won't changing the name affect some visible URLs? E.g. won't the script that downloads the results need a change? If so, then please **don't rename** existing URLs.

Reply:
The script that downloads the result relies on *Workflow* (See https://github.com/enso-org/enso/blob/3a3bef0b46e5dd6d2fd6ce4e27139f2519516e84/tools/performance/engine-benchmarks/bench_tool/__init__.py#L25). The workflow corresponds to a `*.yml` file in the `.github/workflows` directory. This PR does not modify workflows, but it modifies only *jobs*. Rename of *jobs* should not have an impact on gathering the results. But thank you for the observation, I will look into these gathering jobs later and ensure that they are gathering the correct results.

Reply:
Just to be sure, I have created https://github.com/enso-org/enso/issues/9355 as a follow up issue.